rewrite-gradle/src/test/java/org/openrewrite/gradle/UpdateGradleWrapperTest.java

```diff
@@ -664,7 +710,7 @@ void servicesGradleOrgUnavailable() {
           .setHttpSender(unhelpfulSender)
           .setLargeFileHttpSender(unhelpfulSender);
         rewriteRun(
-          spec -> spec.recipe(new UpdateGradleWrapper("8.6", null, null, "https://artifactory.moderne.ninja/artifactory/gradle-distributions/gradle-${version}-bin.zip"))
```

Comment:
I think we can leave this test in-tact just move the version into the wrapperUri

Reply:
To me this test case matters most if using the version lookup from services.gradle. 

If I provide a wrapperURI the fact if the service is available or not should not matter. 
In this case the service is unavailable, we are using services.gradle version and want to bump that version.

With proposed changes, there is no importance of the availability if wrapperUri is provided.